src/pages/Earn/index.tsx

```diff
@@ -72,38 +70,15 @@ export default function Earn() {
     return <ChangeNetworkModal />
   }
 
+  // TODO add info for new users
+
   return (
     <PageWrapper>
-      {stakedFarms.length === 0 && (
-        <TopSection gap="md">
-          <DataCard>
-            <CardNoise />
-            <CardSection>
-              <AutoColumn gap="md">
-                <RowBetween>
-                  <TYPE.white fontWeight={600}>Ubeswap {t('liquidityMining')}</TYPE.white>
-                </RowBetween>
-                <RowBetween>
-                  <TYPE.white fontSize={14}>{t('liquidityMiningDesc')}</TYPE.white>
-                </RowBetween>{' '}
-                <ExternalLink
-                  style={{ color: 'white', textDecoration: 'underline' }}
-                  href="https://docs.ubeswap.org/faq"
-                  target="_blank"
-                >
-                  <TYPE.white fontSize={14}>{t('liquidityMiningReadMore')}</TYPE.white>
-                </ExternalLink>
-              </AutoColumn>
-            </CardSection>
-            <CardNoise />
-          </DataCard>
-        </TopSection>
-      )}
-      <TopSection gap="md">
+      {/* <TopSection gap="md">
         <AutoColumn>
           <TokenSelect onTokenSelect={setFilteringToken} token={filteringToken} />
         </AutoColumn>
-      </TopSection>
+      </TopSection> */}
```

Comment:
should this just be removed?

Reply:
i wasn't 100% sure on this (which is why i left it commented out, but not deleted) - this is for filtering the list of pools displayed by token. but if we are only supporting 5 to begin with, we don't really need this because it will be a short list to scroll through?

Reply:
yeah I vote delete